pkg/extension/workspace/toolchain/builtin/example_config.go

```diff
@@ -101,7 +100,7 @@ func renderExampleConfigTemplate(toolchain *builtinconfig.ToolchainConfig) []byt
 	err = tmpl.Execute(&out, toolchain)
 	if err != nil {
 		// must be caught by unit tests
-		panic(errors.Wrap(err, "failed to render example configuration template"))
+		panic(fmt.Errorf("failed to render example configuration template: %w", err))
```

Comment:
same here

Reply:
can't on this one as it isn't a test. the presence of panics everywhere is more than what we should do in this PR. I cleaned up the ones that were silly (ex the function already returns an error but we chose to panic instead). We can do a later PR to hopefully get rid of all the panics.